test/functional/connection.test.js

```diff
@@ -70,6 +71,21 @@ describe('Connection - functional', function () {
     }
   });
 
+  it('should only pass one argument (topology and not error) for topology "open" events', function (done) {
+    const configuration = this.configuration;
+    const client = configuration.newClient({ w: 1 }, { maxPoolSize: 1 });
+
+    client.on('topologyOpening', () => {
+      client.topology.on('open', (...args) => {
+        expect(args.length).to.equal(1);
```

Comment:
using a more precise assertion will leave a better error message if this fails
```suggestion
        expect(args).to.have.lengthOf(1);
```